CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## v1.1.2
```

Comment:
I was on the fence about the semantic version number to use. This feels like a pretty significant change, but it's not changing the input parameters or anything, only the output. Open to thoughts here, for sure.

Reply:
Thanks both. I think I might opt for a minor version number change because, while this update does change the nature of the values in the temporal dimension, we're not adding/removing dimensions in the final Zarr store. Does that sound like reasonable/sane logic?